workspaces/optic-engine/src/events/http_interaction.rs

```diff
@@ -103,6 +109,20 @@ impl From<&ArbitraryData> for Option<BodyDescriptor> {
   }
 }
 
+impl From<&QueryParametersData> for Option<BodyDescriptor> {
+  fn from(query_param_data: &QueryParametersData) -> Self {
+    let data = &query_param_data.data;
+
+    if let Some(query_string) = &data.as_text {
```

Comment:
What's the intent here?

Reply:
See this other https://github.com/opticdev/optic/pull/937#discussion_r660377516.